.github/ISSUE_TEMPLATE/design_proposal.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Design Proposal
+about: Propose a new design
+title: ''
+labels: proposal
+assignees: ''
+
+---
+
+**What design is being proposed?**
+Supply a summary of the design, perhaps with a problem statement that the proposal is intending to solve.
+
+**Additional Context**
+Provide further details and context around the proposal. What areas of Porter will it affect? (For example: CLI, Manifest/bundle.json, mixins, etc.)
+
+**Risks/Concerns**
+What are some potential concerns with the proposal? (For example: heavy development cost, extensive refactoring, high complexity.)
+
+**What other approaches were considered?**
+Provide other approaches considered prior to deciding on the submitted proposal.
+
+**Implementation details**
+If applicable, provide potential implementation details assuming the proposal is accepted.
+
+_Note that proposals are expected to remain open for at least one week after being created, to allow time for community feedback._
```

Comment:
Should we mention here that proposals should be posted to the mailing list? I think once we have that this is good to go. 👍

Reply:
Oh good call.  Switched this section up to a checklist w/ the addition of the mailing list note.  WDYT?